src/views/design/ScriptDetail/index.tsx

```diff
@@ -256,6 +259,7 @@ const ScriptDetail = withStyles(styles)(
                                     }}
                                     color="secondary"
                                     variant="outlined"
+                                    disabled={!checkAuthority(SECURITY_PRIVILEGES.S_SCRIPTS_WRITE, 'PUBLIC')}
```

Comment:
Can we already use the security group name of the script in question instead of the fixed value 'PUBLIC'.